.pipelines/e2e-tests-template.yml

```diff
@@ -15,7 +15,6 @@ jobs:
         workspace:
           clean: all
         variables:
-          - group: aad-pod-identity
```

Comment:
Won't moving this affect `e2e-tests` for nightly job?

Reply:
Right, originally I put the CODECOV_TOKEN within the `aad-pod-identity` variable group but since I will set it as a secret variable in the PR pipeline.